lib/App/Netdisco/Web/AuthN.pm

```diff
@@ -58,6 +58,7 @@ post '/login' => sub {
 
     if ($success) {
         session logged_in_user => param('username');
+        session logged_in_fullname => schema('netdisco')->resultset('User')->find(param('username'))->get_column('fullname');
```

Comment:
Good idea and thanks for the patch! I just want to see whether we can find a cleaner way of getting the fullname than a complete schema call, here, and then I will be happy to merge.